.github/workflows/lint-and-test.yml

```diff
@@ -0,0 +1,114 @@
+name: lint and test
+
+on: [push, pull_request]
+
+jobs:
+    build-conode:
+        name: build conode
+        runs-on: ubuntu-latest
+        defaults:
+            run:
+                working-directory: conode
+        steps:
+            - uses: actions/checkout@v2
+            - uses: actions/setup-go@v2
+
+            - run: make docker_start_detached docker_wait
```

Comment:
Why does this use two `-run:`? In the other actions there is a
```
run: |
  make...
  docker...
```

Reply:
If find it a bit more readable, especially when reading the resulting build log.